portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,81 @@
+          also learned the cello; however, that had the same ending as the piano. But, now I 
+          am interested in the guitar, especially the classical guitars, and want to learn to 
+          play. Maybe third time is the charm and now that candies are not that important to 
+          me</p>
+        
+      <p>Currently, my family and I reside in Northridge which is in the San Fernando Valley. 
+          Although Cal State Northridge is closer to home, I decided to attend Cal Poly Pomona 
+          to avoid going to the same school as my older sister again and to gain some 
+          independence. Currently, I am an incoming junior studying Computer Science aiming to 
+          graduate in Spring 2024. While my first year and a half of my college experience were 
+          online, I tried to experience the most by taking all the opportunities I can. I joined 
+          the club sheCodes and became the treasurer for 2021-2022 and now the elected vice 
+          president for 2022-2023. While in the club, I participated in 2 Semester Long Projects 
+          (SLP), and 2 outreach programs, Future Women Developers Conference. I also joined side 
+          projects such as the Northrup Grumman Collaboration Projects, Big Data and Cloud 
+          Computing Apprenciship(BDCC), STARs Research project, Target Women in Tech Symposium 
+          Hackathon, etc.</p>
+        
+      <p>For the 2022-2023 school year, I will be studying abroad at Yonsei University in South 
+          Korea. While at Yonsei, I will continue studying computer science and also improve my 
+          Korean language skills. As I advance my knowledge academically, I will also experience 
+          a step-up of my independence and learn how to navigate to survive in a new living 
+          condition for a year. My goal for next year is to become a more knowledgeable person 
+          both in my academic area and personal life! Especially my personal life, because I 
+          still feel like I have not grown much since high school (when covid started).</p>
+        
+      <p>Anywhoo, thank you for reading my about page and hope that you learned one 
+        or two things about me!</p>
+      
+      <h3>Connect with me on:</h3>
+      <ul>
+        <li><a href="https://www.linkedin.com/in/min-jung-70a20013b/">LinkedIn</a></li>
+        <li><a href="https://github.com/minjung1004">GitHub</a></li>
+      </ul>
+     
+      
+      <p>Click <a href="project_list.html">here</a> to see the list of projects I have worked on.</p>
+      
+      <p>Click <a href="gallery.html">here</a> to see more pictures of me!</p>
+      
+     
+      <p>Click here to get a random quote from my favorite TV show Modern Family:</p>
+      <button type='button' onclick="addRandomQuote()">Quote</button>
+      <div id="quote-container"></div>
+
+
+
+     
+
+ <!--<p>Click <a href="/hello">here</a> to see content created by server code.</p>-->
```

Comment:
Generally it's best to remove commented-out code rather than keep it in the file. git will keep the previous revision, and we can always revert back to it later if need be.

Reply:
Edited. I deleted the commented lines!